manifests/sentinel.pp

```diff
@@ -153,8 +153,6 @@
     }
   }
 
-  $sentinel_bind_arr = delete_undef_values([$sentinel_bind].flatten)
```

Comment:
Was removing this intentional?

Reply:
I'm guessing not as the template is using this.